src/util/requirements.ts

```diff
@@ -47,13 +47,6 @@ export interface RequirementsData {
  *
  */
 export async function resolveRequirements(api: JavaExtensionAPI): Promise<RequirementsData> {
-
-    // Use the embedded JRE from 'redhat.java' if it exists
-    const requirementsData = api.javaRequirement;
```

Comment:
How about if we check if this JRE is Java17+?

Reply:
Sure :+1:  
Reinserting